### PR TITLE
Handle multiple orders

### PR DESCRIPTION
**Pull request by GetFridayDevlopment**

### CODE REVIEW

File: dynamo_client.py [hunk 1]
```diff
@@ -23,47 +21,61 @@ def get_customers(self, source_customer_id):
...
-                    'order':esim_order['order'],
-                    'upserted_at': str(datetime.now())
-                })
+            'esim_order_id': esim_order['orderReference'],
+            'order_table_ref_id': order_table_ref.id,
+            'status': esim_order['status'],
+            'currency': esim_order['currency'],
+            'total': esim_order['total'],
+            'order': esim_order['order'],
+            'upserted_at': str(datetime.now())
```
Comment (zaine2003) [inline dynamo_client.py]: just checking that these dates are UTC or derived from the locality of the lambda?

File: lambda_function.py [hunk 2]
```diff
@@ -5,79 +5,102 @@
...
         cust = Customer.from_payload(raw_payload, order)
     else:
         cust = Customer.from_dynamo(existing_customers[0])
         cust.addOrder(order)
 
+    # Save the new customer to DynamoDB
     put_customer_success = dynamo_client.put_customer(cust)
     if not put_customer_success:
-        print("Failed to save new customer:" + cust)
+        print("Failed to save new customer:" + str(cust))
```
Comment (zaine2003) [inline lambda_function.py:37]: is this the full customer object or the ID of that customer? @jamweee are we concerned about PII?

File: dynamo_client.py (hunk 1 above)
Comment (GetFridayDevlopment) [reply dynamo_client.py]: current date and time in the timezone of the Lambda.

File: lambda_function.py (hunk 2 above)
Comment (GetFridayDevlopment) [reply lambda_function.py:37]: full customer object - @zaine2003

Comment (jamweee) [reply lambda_function.py:37]: @GetFridayDevlopment are you able to please check/confirm if this contains any personal information about the customer - such as their name and/or email address? Thanks!

Comment (GetFridayDevlopment) [reply lambda_function.py:37]: I think it does not contain the customer's email and name. we can check it once.

File: dynamo_client.py (hunk 1 above)
Comment (zaine2003) [reply dynamo_client.py]: @jamweee do we need utc?

**Comment by jamweee** [comment]
Yeah I think that makes sense, if it’s not a big effort.

File: lambda_function.py (hunk 2 above)
Comment (GetFridayDevlopment) [reply lambda_function.py:37]: Hi @jamweee checked and we have customer_id, source_name, source_customer_id, orders (order ids), upserted_at.

File: dynamo_client.py (hunk 1 above)
Comment (GetFridayDevlopment) [reply dynamo_client.py]: @zaine2003 changes done -  upserted_at time to UTC

**Review by zaine2003 — APPROVED** [review]

**Pull request merged**